src/plugins/foxPage/components/FoxTab.tsx

```diff
@@ -39,9 +38,8 @@ export const FoxTab: React.FC<FoxTabProps> = ({
       _hover={{ textDecoration: 'none', bg: { base: 'none', md: bgHover } }}
       textAlign='left'
       p={0}
-      isSelected={isSelected}
       onClick={onClick}
-      {...props}
+      {...otherProps}
```

Comment:
We're not passing `isSelected` anymore, so why pass it to `<Tab />` / destructure it in this component?
Not only is it not passed, but it is also undefined.

Reply:
Fixed ! indeed it is totally useless now. Convinced myself that it was still necessary but I probably didn't rebuild correctly the app locally while doing this.